pkg/client/model_create_address.go

```diff
@@ -9,9 +9,10 @@
 
 package client
 
-// CreateCustomerAddress struct for CreateCustomerAddress
-type CreateCustomerAddress struct {
-	Type string `json:"type"`
+// CreateAddress struct for CreateAddress
+type CreateAddress struct {
+	Type      string    `json:"type"`
+	OwnerType OwnerType `json:"ownerType"`
```

Comment:
This breaks the API compatibility for clients and I'm not understanding why the back reference is needed. 

Same question as https://github.com/moov-io/customers/pull/266/files#r514381018

Reply:
I was trying to make it backwards compatible, but I think making the ownerType nullable and using `customer` as the default should make it backwards compatible

Reply:
This should be fixed in my latest commits